subgraphs/smolverse/src/helpers/constants.ts

```diff
@@ -8,7 +8,7 @@ export const SMOL_TREASURES_COLLECTION_NAME = "Smol Treasures";
 export const SWOLERCYCLES_COLLECTION_NAME = "Swolercycles";
 
 // Cron jobs
-export const MISSING_METADATA_UPDATE_INTERVAL = 3600;
+export const MISSING_METADATA_UPDATE_INTERVAL = 600;
```

Comment:
Did this need to be reduced for Swolercycles? I feel like it's going to get triggered a lot with the smols.

Reply:
I think it is okay to be triggered every 10 minutes. If there is no missing metadata then there is no delay. I also changed it to process 100 at a time at max so that on baseURI update it can slowly refresh all the tokens.

Reply:
My early morning brain thought this was every 1 minute. All good!